input/pagecontent/iti-67.md

```diff
@@ -38,11 +38,32 @@ Accept: application/fhir+json
 traceparent: 00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-00
 ```
 
+####  Expected Actions
+
+The Document Responder shall process the query to discover the DocumentReference entries that match the search parameters given.
+
+Document Responder should support search page count requests from the Document Consumer of up to 100 resources.
```

Comment:
Can we have more resources between communities?
The first community has to query all other communities, and fetch all resources, to sort them, before selecting 100 resources to return to the client. This can mean a lot of queries between communities.